core/templates/dev/head/domain/exploration/InteractionObjectFactory.js

```diff
@@ -51,7 +52,7 @@ oppia.factory('InteractionObjectFactory', [
           return hint.toBackendDict();
         }),
         id: this.id,
-        solution: {}
```

Comment:
This needs to be addressed before the PR goes in. If it's easier, you can piggyback on the recent migration I did in #3673 and modify _convert_states_v11_dict_to_v12_dict() -- no need to make a new method since the changes from that PR haven't shipped to production yet. (This note only applies if you address it fast, though -- if this PR for some reason drags on past the next release then a new method must be created.)